src/nspawn/nspawn.c

```diff
@@ -3029,14 +3033,15 @@ static int outer_child(
         if (r < 0)
                 return r;
 
-        if (!cg_ns_supported()) {
+        if (!cg_ns_supported() || !arg_use_cgns) {
```

Comment:
Even if here the order do not matter, I would change it just to be coherent with my comments above.
https://github.com/systemd/systemd/pull/3809/files#r72256570